README.md

```diff
@@ -170,6 +170,7 @@ spec:
     ......   
     ptpSettings:
       stdoutFilter: "^.*master offset.*$"
```

Comment:
add  comments on this what stdoutFilter means and  how logReduce is differ from that